docs/docs/concepts/index.mdx

```diff
@@ -8,17 +8,17 @@ toc: false
 :::info
 
   <p><strong>Contributions are welcome!</strong></p>
-  <p>If there's a document you want to see that's not here, we welcome contributions to add it! Submit a <a href="https://github.com/meltano/meltano/tree/main/docs">pull request</a> with your doc and the Meltano team will help you polish it for release. You may also <a href="https://github.com/meltano/meltano/issues/new">submit an issue</a> to help us gauge interest in new docs.</p>
+  <p>If there's a document you want to see that's not here, we welcome contributions to add it! Submit a <a href="/https://github.com/meltano/meltano/tree/main/docs">pull request</a> with your doc and the Meltano team will help you polish it for release. You may also <a href="/https://github.com/meltano/meltano/issues/new">submit an issue</a> to help us gauge interest in new docs.</p>
```

Comment:
These links are broken now: https://deploy-preview-8211--meltano.netlify.app/concepts/ (assuming just a bad find/replace)

Reply:
Good spot, that was a bad find-replace